### PR TITLE
Ambassadors page

### PR DESCRIPTION
**Pull request by milosyuki**

### CODE REVIEW

File: workspaces/cms-config/src/collections/pages.ts [hunk 1]
```diff
@@ -25,12 +25,24 @@ export const pagesCollectionConfig = {
...
+      label: "Show title",
+      widget: "boolean"
+    },
     {
       name: "template",
       widget: "select",
       options: ["landing", "content"],
       default: "content",
     },
+    {
```
Comment (lorcan-codes) [inline workspaces/cms-config/src/collections/pages.ts:39]: @milosyuki Does CMS allow leaving this field empty while editing? The fields are required by default I think.

File: workspaces/website/src/blocks/AmbassadorsList.tsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+</svg>
+}
+
+const FaAngleLeft = () => {
+  return <svg cursor="pointer" xmlns="http://www.w3.org/2000/svg" width="25" height="24" viewBox="0 0 25 24" fill="none">
+  <path d="M11.7898 12L15.3298 8.46001C15.5161 8.27265 15.6206 8.0192 15.6206 7.75501C15.6206 7.49082 15.5161 7.23737 15.3298 7.05001C15.2369 6.95628 15.1263 6.88189 15.0044 6.83112C14.8825 6.78035 14.7518 6.75421 14.6198 6.75421C14.4878 6.75421 14.3571 6.78035 14.2352 6.83112C14.1134 6.88189 14.0028 6.95628 13.9098 7.05001L9.66982 11.29C9.57609 11.383 9.5017 11.4936 9.45093 11.6154C9.40016 11.7373 9.37402 11.868 9.37402 12C9.37402 12.132 9.40016 12.2627 9.45093 12.3846C9.5017 12.5064 9.57609 12.617 9.66982 12.71L13.9098 17C14.0033 17.0927 14.1141 17.166 14.2359 17.2158C14.3578 17.2655 14.4882 17.2908 14.6198 17.29C14.7514 17.2908 14.8819 17.2655 15.0037 17.2158C15.1256 17.166 15.2364 17.0927 15.3298 17C15.5161 16.8126 15.6206 16.5592 15.6206 16.295C15.6206 16.0308 15.5161 15.7774 15.3298 15.59L11.7898 12Z" fill="black"/>
+  </svg>
+}
```
Comment (lorcan-codes) [inline workspaces/website/src/blocks/AmbassadorsList.tsx]: can we move these two to svg file?

File: workspaces/website/src/pages/(components)/CMSPage.tsx [hunk 3]
```diff
@@ -23,7 +24,7 @@ export default function CMSPage({
   locale,
 }: CMSPageProps) {
   const date = data?.gitlog?.date;
-
+  console.log('data ', data)
```
Comment (lorcan-codes) [inline workspaces/website/src/pages/(components)/CMSPage.tsx]: found a comment to delete ))

**Review by lorcan-codes — CHANGES_REQUESTED** [review]

File: workspaces/website/src/blocks/AmbassadorsList.tsx (hunk 2 above)
Comment (milosyuki) [reply workspaces/website/src/blocks/AmbassadorsList.tsx]: Fixed.

File: workspaces/cms-config/src/collections/pages.ts (hunk 1 above)
Comment (milosyuki) [reply workspaces/cms-config/src/collections/pages.ts:39]: Fixed.

**Review by lorcan-codes — APPROVED** [review]

**Pull request merged**